src/test/resources/jsonld/HT016132962

```diff
@@ -0,0 +1,67 @@
+{
+  "@context" : "http://lobid.org/resources/context.jsonld",
+  "contribution" : [ {
+    "agent" : {
+      "id" : "http://d-nb.info/gnd/16178254-1",
+      "label" : "Konferenz Supply Chain Management im Mittelstand, 2009, Dortmund",
+      "type" : [ "ConferenceOrEvent" ]
+    },
+    "role" : {
+      "id" : "http://id.loc.gov/vocabulary/relators/ctb",
+      "label" : "Mitwirkende"
+    },
+    "type" : [ "Contribution" ]
+  } ],
+  "contributorOrder" : [ "http://d-nb.info/gnd/16178254-1" ],
+  "describedBy" : {
+    "dateCreated" : "20091117",
+    "dateModified" : "20100219",
+    "id" : "http://lobid.org/resources/HT016132962",
+    "label" : "lobid Ressource"
+  },
+  "extent" : "468 S. : Ill., graph. Darst.",
+  "hasItem" : [ {
+    "id" : "http://lobid.org/items/HT016132962:DE-38:15B4921#!",
+    "label" : "lobid Bestandsressource"
+  }, {
+    "id" : "http://lobid.org/items/HT016132962:DE-290:F%2089114-1#!",
+    "label" : "lobid Bestandsressource"
+  }, {
+    "id" : "http://lobid.org/items/HT016132962:DE-6:2E%2022689-1#!",
+    "label" : "lobid Bestandsressource"
+  } ],
+  "hbzId" : "HT016132962",
+  "id" : "http://lobid.org/resources/HT016132962#!",
+  "inSeries" : [ {
+    "numbering" : "1",
+    "series" : [ {
+      "id" : "http://lobid.org/resources/HT016132955#!",
+      "label" : "lobid Ressource"
+    } ],
+    "type" : [ "SeriesRelation" ]
+  } ],
+  "language" : [ {
+    "id" : "http://id.loc.gov/vocabulary/iso639-2/deu",
+    "label" : "Deutsch"
+  } ],
+  "medium" : [ {
+    "id" : "http://rdaregistry.info/termList/RDAproductionMethod/1010",
+    "label" : "Print"
+  } ],
+  "publication" : [ {
+    "location" : "Dortmund",
+    "publishedBy" : "Techn. Univ., Wirtschafts- und Sozialwiss. Fakultät",
+    "startDate" : "2009",
+    "type" : [ "PublicationEvent" ]
+  } ],
+  "sameAs" : [ {
+    "id" : "http://hub.culturegraph.org/resource/HBZ-HT016132962",
+    "label" : "Culturegraph Ressource"
+  } ],
+  "tableOfContents" : [ {
+    "id" : "http://digitool.hbz-nrw.de:1801/webclient/DeliveryManager?pid=4204518&custom_att_2=simple_viewer",
+    "label" : "Digitool"
+  } ],
+  "title" : "Konferenz \"Supply Chain Management im Mittelstand\"",
+  "type" : [ "BibliographicResource", "Book" ]
+}
```

Comment:
Why adding this resource to the test set? Does it test something that is not already tested by an existing test document?